addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/config/EnOceanChannelTotalusageConfig.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal.config;
+
+/**
+ *
+ * @author Dominik Vorreiter - Initial contribution
+ */
+public class EnOceanChannelTotalusageConfig {
+
+    public Boolean validate;
```

Comment:
Please initialize the field directly and use "boolean". No need to use the object wrapper if there is a guaranteed value.

Reply:
Done, thanks!